ubuntu_install_requirements.sh

```diff
@@ -97,7 +98,26 @@ sudo apt -y install \
   python-lxml \
   unzip \
   yarn \
-  genisoimage
+  genisoimage \
+
+
+if [ "${CONTAINER_RUNTIME}" == "podman" ]; then
+  sudo apt -y install podman
+else
+  sudo apt -y install \
+    apt-transport-https \
+    ca-certificates \
+    gnupg-agent \
+    software-properties-common
+  curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+  sudo add-apt-repository \
+    "deb [arch=amd64] https://download.docker.com/linux/ubuntu \
+    $(lsb_release -cs) \
+    stable"
+  sudo apt update
+  sudo apt install -y docker-ce docker-ce-cli containerd.io
+  sudo systemctl start docker
+fi
```

Comment:
This duplicated change should be avoided. I'm also making a patch that both ubuntu and centos installation requirement scripts `source` another script that contains the stuffs which are the same.

Reply:
It is better to separate independent changes in different commits and pull requests to have the smallest changes. Here, adding a new file sourced for one shared line is not justified IMO. Removing duplication is a good idea, but should be done  for the whole Ubuntu and CentOS files in a next commit.